source/query-with-natural-language/ai-and-data-usage-information.txt

```diff
@@ -31,6 +31,12 @@ AI provider:
 - The full text of your natural language prompt.
 - The schema of the collection you are querying, 
   including database name, collection name, field names, and types. 
+- Sample field values. This is an optional setting to improve the 
+  quality of recommendations. You can manage this through the
+  ``Enable sending sample field values with query and aggregation 
+  generation requests`` setting in 
+  :ref:`Compass settings <compass-settings-reference>`. This setting 
```

Comment:
Not a blocker, but this does create a scrollbar on my screen size.
<img width="906" alt="Screenshot 2024-06-12 at 9 22 57 AM" src="https://github.com/mongodb/docs-compass/assets/1791149/a4295437-d4f8-4748-97ab-a27168bd2830">

Reply:
Yeah I wasn't sure how to handle this, as we typically monospace UI elements. Is it possible to shrink the setting name as its the largest setting name.

Spitballing:
- Enable sending sample field values with AI requests
- Enable sending sample field values with Generative AI
- Enable sending sample field values 

The description below in the settings pane describes the setting and it already sits in the `Artificial Intelligence` tab, so I am not sure the name needs to be so long.

![Screenshot 2024-06-12 at 12 46 49 PM](https://github.com/mongodb/docs-compass/assets/85948430/a3c9241d-125c-4b76-bebc-b578d2d316bc)


@Anemy